stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/Events.kt

```diff
@@ -0,0 +1,90 @@
+package io.getstream.chat.docs.kotlin
+
+import io.getstream.chat.android.client.ChatClient
+import io.getstream.chat.android.client.channel.ChannelClient
+import io.getstream.chat.android.client.channel.subscribeFor
+import io.getstream.chat.android.client.events.ChatEvent
+import io.getstream.chat.android.client.events.ConnectedEvent
+import io.getstream.chat.android.client.events.ConnectingEvent
+import io.getstream.chat.android.client.events.DisconnectedEvent
+import io.getstream.chat.android.client.events.NewMessageEvent
+import io.getstream.chat.android.client.events.NotificationAddedToChannelEvent
+import io.getstream.chat.android.client.events.UserPresenceChangedEvent
+import io.getstream.chat.android.client.subscribeFor
+import io.getstream.chat.android.client.subscribeForSingle
+import io.getstream.chat.android.client.utils.observable.Disposable
+
+class Events(val client: ChatClient, val channelController: ChannelClient) {
+
+    fun listenSpecificChannelEvents() {
+        // Subscribe for new message events
+        val disposable: Disposable = channelController
+            .subscribeFor<NewMessageEvent> { newMessageEvent ->
+                // to get the message
+                val message = newMessageEvent.message
+            }
+
+        // Dispose when you want to stop receiving events
+        disposable.dispose()
+    }
+
+    fun listenAllChannelEvents() {
+        val disposable: Disposable = channelController
+            .subscribe { event: ChatEvent ->
+                when (event) {
+                    is NewMessageEvent -> {
+                        // to get the message
```

Comment:
I'd decide on either lower- or uppercase comments, and use the same style everywhere here, since it's going into the documentation. My vote is starting comments with uppercase.